bst_plugin/process_nst_compute_ttest.m

```diff
@@ -228,15 +228,19 @@
 
 %         % Saving the cB Matrix
     sOutput_b = db_template('matrixmat');
-    sOutput_b.Value           = B.Value';
+    sOutput_b.F           = B.Value';
     sOutput_b.Comment     = [contrast ' B' ];
```

Comment:
I think we should prefix all the GLM-related outputs with 'GLM_'.
A condition folder can get quickly get really full of various items.